src/libraries/Microsoft.PowerFx.Core/Public/DisplayNames/DisplayNameUtility.cs

```diff
@@ -23,7 +23,7 @@ public static class DisplayNameUtility
         /// </summary>
         /// <param name="logicalToDisplayPairs">Enumerable of (logical, display) pairs.</param>
         /// <returns>Enumerable of unique (logical, display) pairs.</returns>
-        public static IEnumerable<KeyValuePair<DName, DName>> MakeUnique(IEnumerable<KeyValuePair<string, string>> logicalToDisplayPairs)
+        public static DisplayNameProvider MakeUnique(IEnumerable<KeyValuePair<string, string>> logicalToDisplayPairs)
```

Comment:
This is a breaking change to a public method

Reply:
Seems reasonable since its preview